Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -98,9 +100,9 @@ public virtual AckStrategy HandleConsumerError(ConsumerExecutionContext context,
 
                 model.BasicPublish(errorExchange, receivedInfo.RoutingKey, errorProperties, message.Memory);
 
-                if (!configuration.PublisherConfirms) return AckStrategies.Ack;
+                if (!configuration.PublisherConfirms) return Task.FromResult(AckStrategies.Ack);
```

Comment:
@Pliner All changes for `Task.FromResult` in this file now produce new memory allocations. It would be nice to move all these tasks into static readonly fields.